gradle.properties

```diff
@@ -1,9 +1,6 @@
 projectVersion=9.0.0-SNAPSHOT
 #4.0.2-4.0.3 breaks org.ysb33r.gradle:grolifant for chromedriver
 asciidoctorGradleVersion=4.0.1
```

Comment:
Grails-data-mapping & gorm-hibernate5 are meant to be used outside of Grails applications.  Are any of these versions still required to set version properties correctly when grails-bom isn't in use?  

I'm assuming the jakarta version due to the older hibernate import may be the only one?  The rest look like tests or grails specific ones.

Reply:
Yes, all remaining versions in `gradle.properties` are still used somewhere in the project.

This PR (so far) only aims to clean up versions from the test/example apps by using the versions listed in the `grails-bom`.
(I'm sure we can cleanup the project even further down the line.)